pkg/expr/threshold.go

```diff
@@ -18,23 +18,32 @@ import (
 type ThresholdCommand struct {
 	ReferenceVar  string
 	RefID         string
-	ThresholdFunc string
+	ThresholdFunc ThresholdFunc
 	Conditions    []float64
 	Invert        bool
 }
 
+// The reducer function
+// +enum
+type ThresholdFunc string
```

Comment:
maybe "ThresholdID"?  but this was the name of the string variable, so i just used it